spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CommentsController do
+  let(:user) { create(:learner) }
+  let(:rfc_with_comment) { create(:rfc_with_comment, user: user) }
+  let(:comment) { rfc_with_comment.comments.first }
+  let(:updated_comment) { comment.reload }
+  let(:perform_request) { proc { put :update, format: :json, params: {id: comment.id, comment: comment_params} } }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    perform_request.call
+  end
+
+  describe 'PUT #update' do
+    context 'with valid params' do
+      let(:comment_params) { {text: 'test100'} }
+
+      it 'saves the permitted changes' do
+        expect(updated_comment.text).to eq('test100')
+      end
+
+      expect_http_status(:ok)
+    end
+
+    context 'with invalid params' do
```

Comment:
Wording might not be perfect here, but I'd say it's good enough.

```
context 'with invalid params' do
let(:comment_params) { {text: 'test100', row: 5, file_id: 50} }

      it 'applies the permitted changes' do
```

Reply:
It's rather `'with additional params'`.

Reply:
Thanks, I made the adjustment and will proceed with merging once the pipeline passes.